src/Parser.zig

```diff
@@ -137,9 +137,13 @@ fn checkIdentifierCodepoint(comp: *Compilation, codepoint: u21, loc: Source.Loca
 
 fn eatIdentifier(p: *Parser) !?TokenIndex {
     if (p.tok_ids[p.tok_i] == .identifier) {
+        if (mem.indexOf(u8, p.tokSlice(p.tok_i), "$")) |_|
```

Comment:
Can use `indexOfScalar` (with `'$'`) instead of `indexOf`

Reply:
Done in recent commit